install.sh

```diff
@@ -15,6 +15,12 @@ echo
 curl -sL $DOWNLOAD_URL -o $OUTPUT_BASENAME_WITH_POSTFIX
 echo -e "\033[32m[V] Downloaded Datree"
 
+if ! unzip >/dev/null 2>&1;then
+    echo -e "\e[1;31m error: unzip command not found \e[0m"
```

Comment:
The output in the terminal doesn't parse the formatting properly.

<img width="432" alt="Screen Shot 2021-10-05 at 10 35 49" src="https://user-images.githubusercontent.com/1208902/135980522-1e8afa23-621a-44e4-9edb-4d462a579737.png">
)